src/__tests__/local-state/general.ts

```diff
@@ -819,6 +830,102 @@ describe('Combining client and server state/operations', () => {
     }, 10);
   });
 
+  itAsync.only('query resolves with loading: false if subsequent responses contain the same data', (resolve, reject) => {
```

Comment:
```suggestion
  itAsync('query resolves with loading: false if subsequent responses contain the same data', (resolve, reject) => {
```

You probably don't want this to ensure other tests in this file run as well :)

Reply:
Oops - meant to revert this before pushing, thanks!